projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -84,55 +97,195 @@ export class IgxTimePickerComponent implements ControlValueAccessor, OnInit, OnD
         }
     }
 
+    /**
+     * An accessor that returns the value of the `_value` property.
+     * ```html
+     *@ViewChild("MyPick")
+     *public pick: IgxTimePickerComponent;
+     *ngAfterViewInit(){
+     *    let pickSelect = this.pick.value;
+     * }
+     * ```
+     */
     get value(): Date {
         return this._value;
     }
 
+    /**
+     * An @Input property that allows you to disable the component. By default it's not applied.
```

Comment:
Instead of 'By default it's not applied' use: By default 'isDisabled is set to false'